src/effects/effectchainslot.cpp

```diff
@@ -362,28 +887,36 @@ void EffectChainSlot::slotControlClear(double v) {
 }
 
 void EffectChainSlot::slotControlChainEnabled(double v) {
-    //qDebug() << debugString() << "slotControlChainEnabled" << v;
-    if (m_pEffectChain) {
-        m_pEffectChain->setEnabled(v > 0);
-    }
+    qDebug() << debugString() << "slotControlChainEnabled" << v;
+    // NOTE(Kshitij) : Removed the reference to m_pEffectChain
+    // if (m_pEffectChain) {
+    //     m_pEffectChain->setEnabled(v > 0);
+    // }
+    // if (m_pEffectChain) {
+    setEnabled(v > 0);
+    // }
 }
 
 void EffectChainSlot::slotControlChainMix(double v) {
-    //qDebug() << debugString() << "slotControlChainMix" << v;
+    qDebug() << debugString() << "slotControlChainMix" << v;
 
     // Clamp to [0.0, 1.0]
     if (v < 0.0 || v > 1.0) {
         qWarning() << debugString() << "value out of limits";
         v = math_clamp(v, 0.0, 1.0);
         m_pControlChainMix->set(v);
     }
```

Comment:
ControlPotmeter makes sure the value stays in bounds already, so this code is redundant.